src/employees/flat-employees.ts

```diff
@@ -0,0 +1,6 @@
+import { Employee } from './employee.interface';
+import { data } from './data';
+import { flattenData } from './flatten-data';
+
+export const flatEmployees: Array<Employee> = flattenData(data);
```

Comment:
Executing the function in-line has two drawbacks:
* The result is shared across _all importers_.
* The code is executed even if nobody imports it.

A function that creates the data set on demand does not have these problems:
```ts
// src/employees/data.ts
export const getEmployees = () => parseData(rawData);
...
// src/employees/flat-employees.ts
export const getFlatEmployees = () => flattenData(getEmployees());
```